deployment/flexvol-installer/install.sh

```diff
@@ -20,5 +20,5 @@ cp /bin/azurekeyvault-flexvolume ${kv_vol_dir}/azurekeyvault-flexvolume #script
 #https://github.com/kubernetes/kubernetes/issues/17182
 # if we are running on kubernetes cluster as a daemon set we should
 # not exit otherwise, container will restart and goes into crashloop (even if exit code is 0)
-while true; do echo "install done, daemonset sleeping" && sleep 300; done
+while true; do echo "install done, daemonset sleeping" && sleep 60; done
```

Comment:
yes, this has nothing to do with keyvault, it simply makes the daemonset container alive so it won't be restarted by K8s.

honestly I would even reduce it it to 5-10 seconds, so it will allow faster termination for sigterm.